.github/workflows/ScreenShotTest.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    branches:
+      - main
```

Comment:
This may not be necessary as it is only for pull requests.

Reply:
You're absolutely right.